entrypoint.sh

```diff
@@ -1,3 +1,37 @@
-#!/bin/sh -l
+#!/bin/bash
 
-/go/bin/tfsec /github/workspace
+TFSEC_OUTPUT=$(/go/bin/tfsec /github/workspace)
+TFSEC_EXITCODE=${?}
+
+# Exit code of 0 indicates success.
+if [ ${TFSEC_EXITCODE} -eq 0 ]; then
+  TFSEC_STATUS="Success"
+else
+  TFSEC_STATUS="Failed"
+fi
+
+# Print output.
+echo "${TFSEC_OUTPUT}"
+
+# Comment on the pull request if necessary.
+if [ $INPUT_TFSEC_ACTIONS_COMMENT == "1" ] || [ $INPUT_TFSEC_ACTIONS_COMMENT == "true" ]; then
```

Comment:
```suggestion
if [ "$INPUT_TFSEC_ACTIONS_COMMENT" == "1" ] || [ "$INPUT_TFSEC_ACTIONS_COMMENT" == "true" ]; then
```

Double quote to prevent globbing and word splitting. https://github.com/koalaman/shellcheck/wiki/SC2086